### PR TITLE
Fixing a few broken docs

### PR DESCRIPTION
**Pull request by amyblais**
Also adjusting one of the upgrade notes for 5.37.

### CODE REVIEW

File: source/messaging/managing-channels.rst [hunk 1]
```diff
@@ -143,7 +143,10 @@ To unmute the channel, select the channel name again to access the drop-down men
 Switching channels
 ------------------
 
-Use the **Find channel** option in the channel sidebar, or press CMD/CTRL+K, to visit other channels, or to review `member availability <https://docs.mattermost.com/help/getting-started/setting-your-status-availability.html#setting-your-availability>`__ at a glance. 
+Use the **Find channel** option in the channel sidebar, or press CMD/CTRL+K, to visit other channels, or to review `member availability <https://docs.mattermost.com/help/getting-started/setting-your-status-availability.html#setting-your-availability>`__ at a glance.
+
+    .. image:: ../../images/switch-channels.png
```
Comment (amyblais) [inline source/messaging/managing-channels.rst:148]: Not sure if this image was intentionally removed, but it doesn't work in preview.

PR for reference: https://github.com/mattermost/docs/pull/4698/files

Comment (cwarnermm) [reply source/messaging/managing-channels.rst:148]: @amyblais Image links need to be updated to ``../images/filename.png`` rather than ``../../images``

Comment (cwarnermm) [inline source/messaging/managing-channels.rst:148]:
```suggestion
    .. image:: ../images/switch-channels.png
```

Comment (cwarnermm) [inline source/messaging/managing-channels.rst]:
```suggestion
    .. image:: ../images/switch-channels.png
```

Comment (cwarnermm) [reply source/messaging/managing-channels.rst]: I've made these updates.

File: source/install/self-managed-changelog.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ Lastest Mattermost Releases:
 **Release Day: 2021-07-16**
 
 ### Deprecations
- - The ``platform`` binary and “--platform” flag have been deprecated. If you are using the “--platform” flag or are using the ``platform`` binary directly to run the Mattermost server application via a systemd file or custom script, you will be required to use only the mattermost binary.
+ - The ``platform`` binary and “--platform” flag have been deprecated.
```
Comment (streamer45) [inline source/install/self-managed-changelog.md:21]: I was actually thinking of replacing the word "deprecated" in favor of "removed". The rest of the message is fine. 
I guess the point I'd like to pass is that this is not a simple deprecation warning. That went out at the begin of the year if I remember correctly. Now we acted on the deprecation and completely removed the binary.

**Comment by cwarnermm** [comment]
@amyblais - Users who click on existing docs links with now-outdated URLs will be automatically redirected to the new page. We've been updating the older links as we encounter them, but we included the 301 redirects too to assist.

**Review by cwarnermm — APPROVED** [review]
Updated the image file paths

**Review by streamer45 — APPROVED** [review]
Awesome, thanks Amy :+1:

**Pull request merged**